canonical-websites/www.ubuntu.com
# lp - multi-cloud guide

## Done

* New landing page at `/engage/multi-cloud-guide`
* [copy doc ](https://docs.google.com/document/d/1fjt7cKzopJk6GPWKNQiUU3fv9p4yZ1OCcVUW1_Q2NrU/edit#)

## TODO

* [x]  ~**Update form ID when available**~
* [x]  ~**Update form return URL when available**~
* [x] Update hero with takeover design

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure the page looks good and matches the copy doc
- Ensure the form takes you to the right thank you page
